pkg/catalog/install_test.go

```diff
@@ -67,38 +67,37 @@ var _ = Describe("Install", func() {
 				Version:       "v0.0.1",
 			},
 			Clients: catalog.Clients{
-				CatalogClient: fakeCatalogClient,
-				GitClient:     fakeGit,
+				CatalogClient:  fakeCatalogClient,
+				GitClient:      fakeGit,
+				ArtifactsMaker: fakeMakeArtifacts,
 			},
 			Directory: tempDir,
 		}
-		fakeMakeArtifacts = func(sub profilesv1.ProfileInstallation, gitClient git.Git, rootDir, gitRepoNamespace string, gitRepoName string) ([]profile.Artifact, error) {
-			return []profile.Artifact{
-				{
-					Objects: []runtime.Object{
-						&kustomizev1.Kustomization{
-							ObjectMeta: metav1.ObjectMeta{
-								Name:      "foo",
-								Namespace: "default",
-							},
-							TypeMeta: metav1.TypeMeta{
-								Kind:       "kustomize",
-								APIVersion: "api",
-							},
-							Spec: kustomizev1.KustomizationSpec{
-								Prune: true,
-							},
+		fakeMakeArtifacts.MakeArtifactsReturns([]profile.Artifact{
+			{
+				Objects: []runtime.Object{
+					&kustomizev1.Kustomization{
+						ObjectMeta: metav1.ObjectMeta{
+							Name:      "foo",
+							Namespace: "default",
+						},
+						TypeMeta: metav1.TypeMeta{
+							Kind:       "kustomize",
+							APIVersion: "api",
+						},
+						Spec: kustomizev1.KustomizationSpec{
+							Prune: true,
 						},
 					},
-					Name: "foo",
 				},
-			}, nil
-		}
+				Name: "foo",
+			},
+		}, nil)
 	})
 
-	JustBeforeEach(func() {
-		catalog.SetMakeArtifacts(fakeMakeArtifacts)
-	})
+	//JustBeforeEach(func() {
+	//	catalog.SetMakeArtifacts(fakeMakeArtifacts)
+	//})
 
```

Comment:
?

Reply:
ops, left that in there, I want to delete it, thanks